_rules/audio-transcript-2eb176.md

```diff
@@ -24,12 +31,12 @@ acknowledgments:
 
 This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- autoplaying; or,
+- has a [play button][] that is [visible][].
```

Comment:
I do not really agree with the removal of the inclusion in the accessibility tree.
Looking at the survey report, the argument was that this might be a 1.3.1 failure rather than a 1.2.1. However, looking at the [Understanding document](https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded.html) (which is not normative, but still very helpful) (emphasizes mine):
* The first sentence of the Intent is:
  > The intent of this Success Criterion is to make information conveyed by prerecorded audio-only and prerecorded video-only content **available to all users**.

  "available to all users" does imply that it is in the accessibility tree, since it would otherwise not be available to AT users.
  Especially, if a silent video has a hidden transcript, this does not provide any kind of alternative for blind users and I think that would fail 1.2.1.
* The first Benefit is:
  > This Success Criterion helps people who have **difficulty perceiving visual content**. Assistive technology can read text alternatives aloud, present them visually, or **convert them to braille**.

  People who have difficulties with visual content would not benefit at all from an alternative that is not included in the accessibility tree; as far as I know, braille displays also work with the accessibility tree and won't be able to present the alternative.
* The fourth benefit is:
  > People who are deaf-blind can read the text in braille.

  Again, not sure that this is possible if the alternative is not included in the accessibility tree.

I'm rather of the opinion that having an inaccessible alternative is actually a failure of 1.2.1 and thus should be part of this rule. This is maybe something that we need to discuss further.